board/arm-64bit-toolchain.cmake

```diff
@@ -0,0 +1,52 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR ARM)
+
+if(MINGW OR CYGWIN OR WIN32)
+    set(UTIL_SEARCH_CMD where)
+elseif(UNIX OR APPLE)
+    set(UTIL_SEARCH_CMD which)
+endif()
+
+## You need to add the full path in TOOLCHAIN_PREFIX, if you do not use the
+## standard gcc toolchain 
+set(TOOLCHAIN_PREFIX aarch64-linux-gnu-)
+
+## In some distribution, a dynamic link for aarch64-linux-gnu-gcc may not be
+## found or created, instead it might be labelled with the version at the end
+## For instance: aarch64-linux-gnu-gcc-5
+## Therefore, if dynamic link exists, you do not have to specify the version
+set(VERSION_NUMBER "" CACHE STRING "Enter the version number of the compiler")
+
+# Without that flag CMake is not able to pass test compilation check
+set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+
+set(CMAKE_AR "${TOOLCHAIN_PREFIX}gcc-ar${VERSION_NUMBER}" CACHE FILEPATH "" FORCE)
+set(CMAKE_C_COMPILER ${TOOLCHAIN_PREFIX}gcc${VERSION_NUMBER})
+set(CMAKE_CXX_COMPILER ${TOOLCHAIN_PREFIX}g++${VERSION_NUMBER})
+set(CMAKE_LINKER ${TOOLCHAIN_PREFIX}ld${VERSION_NUMBER})
+set(CMAKE_C_ARCHIVE_CREATE "<CMAKE_AR> qcs <TARGET> <LINK_FLAGS> <OBJECTS>")
+set(CMAKE_C_ARCHIVE_FINISH  true)
+set(CMAKE_FORTRAN_COMPILER ${TOOLCHAIN_PREFIX}gfortran)
+set(CMAKE_ASM_COMPILER ${CMAKE_C_COMPILER})
+set(CMAKE_OBJCOPY ${TOOLCHAIN_PREFIX}objcopy${VERSION_NUMBER} CACHE INTERNAL "objcopy tool")
+set(CMAKE_SIZE_UTIL ${TOOLCHAIN_PREFIX}size${VERSION_NUMBER} CACHE INTERNAL "size tool")
+
+## There is no need to specify the CMAKE_SYSROOT if you are using the 
+## standard toolchain. If you download your own toolchain you have to specify
+## the path for sysroot as follows:
+## set(CMAKE_SYSROOT /PathToToolchain/aarch64-buildroot-linux-gnu/sysroot)
+## or it can be specified from command-line.
+set(CMAKE_SYSROOT /usr/aarch64-linux-gnu/ CACHE STRING "Enter path for sysroot")
+
+## Here are the standard ROOT_PATH if you are using the standard toolchain
+## if you are using a different toolchain you have to specify that too
+set(CMAKE_FIND_ROOT_PATH
+  /usr/aarch64-linux-gnu/ 
```

Comment:
To make this easier on users, you could either set it to `${CMAKE_SYSROOT}`, or maybe you might be able to remove it too.  I'm using this as an example:

https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html#cross-compiling-for-linux

and they don't specify `CMAKE_FIND_ROOT_PATH` there, so maybe we can get away without it?